packages/fields/package.json

```diff
@@ -49,12 +49,14 @@
     "inflection": "^1.12.0",
     "intersection-observer": "^0.5.1",
     "is-hotkey": "^0.1.4",
+    "lodash.get": "^4.4.2",
     "lodash.groupby": "^4.6.0",
     "lodash.isequal": "^4.5.0",
     "lodash.omitby": "^4.6.0",
     "luxon": "^1.11.4",
     "memoize-one": "^4.0.2",
     "mongoose": "^5.5.2",
+    "p-is-promise": "^2.1.0",
```

Comment:
If we're adding to package.json we should update our yarn.lock as well.

Reply:
It's already there (I think it was brought in via a transitive dependency) :+1: